app/assets/javascripts/editor.js

```diff
@@ -1,13 +1,16 @@
-// jQuery (document).ready function:
-
-$E = {
-  initialize: function() {
-    // call setState with no parameters, aka. default parameters.
-    // default parameters point toward either:
-    //   1. the comment form at the bottom of multi-comment wikis/questions/research notes
-    //   2. the only editor form on /wiki/new and /wiki/edit
-    $E.setState();
-    
+class Editor {
+  // default parameter references the ID of:
+  //   1. main comment form in multi-comment wikis, questions, & research notes.
+  //   2. the only editor form on /wiki/new and /wiki/edit
+  constructor(commentFormID = "main") {
```

Comment:
There will be merge conflicts after we merge this PR: https://github.com/publiclab/plots2/pull/9104/files